vision_project/predict.py

```diff
@@ -0,0 +1,193 @@
+import os
```

Comment:
this is not my file, I rebased my changes onto main. lets make sure before we merge that this file is fine